samples/drivers/gpio/devices/mcp23x17_spi.overlay

```diff
@@ -23,9 +29,9 @@
 &spi0 {
 	status = "ok";
 
-	gpioex0:mcp23x17@20 {
+	gpioex0:mcp23x17@40 {
```

Comment:
The `reg` seems wrong; is this really using the 65th chip-select supported by this SPI bus?

Reply:
Sorry - that's a remnant from some testing I did.
I'll change it back to reflect the actual range of the chip (0x20 - 0x27).

Reply:
No.  The reg field on a SPI device identifies the chip select ordinal on the SPI bus.  It's completely unrelated to the I2C address.  You probably want 0 here.

But more importantly: where are these overlays used?  The sample that they were attached to was removed in bc6a0152350aca6476812247f3d0f82c3643338f over a year ago.

You should just remove these two overlays.